applications/GeoMechanicsApplication/custom_constitutive/coulomb_yield_surface.hpp

```diff
@@ -0,0 +1,42 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//
+//  Main authors:    Mohame Nabi,
+//                   Wijtze Pieter Kikstra
+//
+
+#pragma once
+
+#include "custom_constitutive/yield_surface.h"
+
+namespace Kratos
+{
+
+class KRATOS_API(GEO_MECHANICS_APPLICATION) CoulombYieldSurface : public YieldSurface
+{
+public:
+    KRATOS_CLASS_POINTER_DEFINITION(CoulombYieldSurface);
+
+    CoulombYieldSurface() = default;
+
+    CoulombYieldSurface(double frictionAngle, double cohesion, double dilatationAngle);
```

Comment:
The function parameter names don't follow the Kratos Style Guide:
```suggestion
    CoulombYieldSurface(double FrictionAngle, double Cohesion, double DilatationAngle);
```

Also, for clarity, you may want to add the expected angular units to the function parameter names, e.g. `FrictionAngleInRad`. The same applies to the corresponding data members (`mFrictionAngle` and `mDilatationAngle`).

Reply:
I still miss the angular unit in the variable names. Could you please add that? Thanks.